packages/titanium-data-table/src/titanium-data-table-header.ts

```diff
@@ -5,7 +5,7 @@ export class TitaniumDataTableHeaderElement extends LitElement {
   @property() title: string;
   @property() sortBy: string;
   @property({ reflect: true, type: String }) width: string;
-  @property({ reflect: true, type: Boolean }) active: Boolean;
+  @property({ reflect: true, type: Boolean }) active: Boolean = false;
```

Comment:
dont need this right?

Reply:
I set it to false to be explicit, otherwise it will be undefined which is also falsey.  So no it isn't needed....   You want me to remove it?